src/nfa/nfa-inclusion.cc

```diff
@@ -225,10 +281,11 @@ bool Mata::Nfa::is_incl(
 { // {{{
     AlgoType algo{ set_algorithm(std::to_string(__func__), params) };
 	return algo(smaller, bigger, alphabet, cex, params);
-} // is_incl }}}
+} // is_included }}}
 
 bool Mata::Nfa::are_equivalent(const Nfa& lhs, const Nfa& rhs, const Alphabet *alphabet, const StringMap& params)
 {
+    //TODO: add comment on what this is doing, what is __func__ ...
```

Comment:
`__func__` is afaik preprocessor "constant" that is translated to the name of the current function (similar to `__line__` (current line in code) or `__file__` (current file).